account-kit/react/src/components/auth/card/error/types.ts

```diff
@@ -15,10 +12,11 @@ export enum EOAWallets {
 }
 
 export type ConnectionErrorProps = {
```

Comment:
oh didn't realize this was here, we can put this and export it from the component file itself. it's what we do in most components already. we might not even need to export it explicitly if it's not used elsewhere to be extended

Reply:
sounds good, removed this file and collocated the types